helm/charts/hydra/templates/_helpers.tpl

```diff
@@ -82,6 +82,18 @@ Generate the name of the secret resource containing secrets
 {{- end -}}
 {{- end -}}
 
+{{/*
+Generate the secrets.annotations value
+*/}}
+{{- define "hydra.secrets.annotations" -}}
+  {{- $annotations := dict -}}
+  {{- if .Values.secret.enableDefaultAnnotations }}
+    {{- $annotations = .Values.secret.secretAnnotations -}}
+  {{- end -}}
+  {{- $annotations = merge $annotations .Values.secret.extraAnnotations -}}
+  {{- toYaml $annotations }}
```

Comment:
i think we could just take the extraAnnotations and do the conditional merge to it, but it's programmatically the same :)